{{cookiecutter.project_slug}}/README.md

```diff
@@ -81,3 +91,34 @@ exec fastlane match development` from `ios/`.
 Not a CB developer? Create an [Apple developer](https://developer.apple.com)
 and follow the instructions on [codesigning guide](https://codesigning.guide/)
 to setup your certificates.
+
+##Blueprints
+
+###Google Maps API
+
+As soon as the project is created, you need to run:
+
+`npm install react-native-maps --save-exact`
+`yarn add react-native-maps -E`
+
+###Camera
+
+Do:
+
+`npm install react-native-camera --save`
```

Comment:
we don't need the developer to install this dependency, we can just add it into pakcage.json :-)